src/Compilers/CSharp/Portable/FlowAnalysis/CSharpDataFlowAnalysis.cs

```diff
@@ -372,5 +389,10 @@ private static ImmutableArray<ISymbol> Normalize(IEnumerable<Symbol> data)
         {
             return ImmutableArray.CreateRange(data.Where(s => s.CanBeReferencedByName).OrderBy(s => s, LexicalOrderSymbolComparer.Instance).GetPublicSymbols());
         }
+
+        private static ImmutableArray<IMethodSymbol> Normalize(IEnumerable<MethodSymbol> data)
+        {
+            return ImmutableArray.CreateRange(data.Where(s => s.CanBeReferencedByName).OrderBy(s => s, LexicalOrderSymbolComparer.Instance).Select(p => p.GetPublicSymbol()));
```

Comment:
>Select(p => p.GetPublicSymbol()) [](start = 140, length = 32)

There's a `GetPublicSymbols()` extension that can be used here.

Reply:
Only if I call `ToImmutableArray` after the `OrderBy`. Doesn't seem worth it to me, but I could be wrong...

Reply:
It seems to work in the method right before this one :)

Reply:
As discussed, that is this, and its the only extension on IEnumerable. Happy to add another, I have no context on whether this scenario is a common one
https://github.com/dotnet/roslyn/blob/master/src/Compilers/CSharp/Portable/Symbols/SymbolExtensions.cs#L661